extensions/industry/healthcare/fhir/fieldgroups/medication.schema.json

```diff
@@ -15,7 +15,8 @@
       ]
    },
    "meta:intendedToExtend":[
-      "https://ns.adobe.com/xdm/classes/medication"
+      "https://ns.adobe.com/xdm/classes/medication",
+      "https://ns.adobe.com/xdm/context/profile"
```

Comment:
Does it need to be allowed on the profile schema?

Reply:
Yes, Having only **medication** is limiting the use. 
Addition of profile class is approved by @digant!